lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -214,21 +214,21 @@ class TestFieldOverrideMongoPerformance(FieldOverridePerformanceTestCase):
 
     TEST_DATA = {
         # (providers, course_width, enable_ccx, view_as_ccx): # of sql queries, # of mongo queries, # of xblocks
-        ('no_overrides', 1, True, False): (48, 6, 13),
-        ('no_overrides', 2, True, False): (135, 6, 84),
-        ('no_overrides', 3, True, False): (480, 6, 335),
-        ('ccx', 1, True, False): (48, 6, 13),
-        ('ccx', 2, True, False): (135, 6, 84),
-        ('ccx', 3, True, False): (480, 6, 335),
+        ('no_overrides', 1, True, False): (49, 6, 13),
```

Comment:
Is there any way to avoid the extra SQL call on every page? Is this to fetch the user's current language preference? I wonder if we are already looking this up to set up the request context, in which case it could be cached or accessed from the context. @ormsbee any suggestions?